install/scripts/docker_entrypoint.sh

```diff
@@ -46,6 +46,9 @@ RUN_TEST=false
 SQL_HOST=127.0.0.1
 SQL_PORT=3306
 
+SERVER_HOST=127.0.0.1
```

Comment:
I recall that Duncan before had issues with `127.0.0.1`, and he can only get things working by using `0.0.0.0`. Is there any concerns here?

Reply:
That's a fair question.  I can change it but I don't think it's an issue.  In the Helm chart, I pass in the right address in the case of the server

https://github.com/kevensen/helm-charts/blob/866ef8f53af8d234a001575c8ae07d4c613afe4a/charts/forseti-security/templates/server/deployment.yaml#L46-L49

And the right hostname in the case of the orchestrator:

https://github.com/kevensen/helm-charts/blob/866ef8f53af8d234a001575c8ae07d4c613afe4a/charts/forseti-security/templates/orchestrator/forseti-cron-job.yaml#L37-L40